app.js

```diff
@@ -0,0 +1,14 @@
+const express = require('express')
+const { getTopics } = require('./controllers/topics.controllers')
+const { handleMissingEndpoints, handleServerErrors } = require('./controllers/error.controllers')
+
+app = express()
+
+app.get('/api/topics',getTopics)
+app.get('',)
+
+app.use(handleMissingEndpoints)
+app.use(handleServerErrors)
+
+
+module.exports = {app}
```

Comment:
Nothing wrong with this export, however highly unlikely that you'll be exporting more than app in this example.